dev/setup-cluster-config

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
Should this warn for uninitialized variables? 
```suggestion
set -exu
```

Reply:
Should not. "$1" is optional and there are other optional environment variables in it as well.